.github/workflows/rust.yml

```diff
@@ -55,3 +55,24 @@ jobs:
     - uses: actions/checkout@v2
     - name: Run cargo fmt
       run: cargo fmt --all -- --check
+
+  publish:
+    runs-on: ubuntu-20.04
+    needs: [tests, clippy, cargo-fmt]
+    if: github.ref == 'refs/heads/master'
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup
+      run: |
+        git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        cargo install --git https://github.com/miraclx/cargo-workspaces --branch grouping-versioning-and-exclusion cargo-workspaces
```

Comment:
This feels a bit sketchy, as the git branch is mutable. I think this is OK, but we probably should publish a proper `near-cargo-workspaces`, and use `cargo install near-cargo-workspace --version x.y.z --locked`. Doesn't need to block the PR though.

Reply:
Valid point, in the meantime, I suppose we could use `--rev` and specify a git revision. Wdyt?